### PR TITLE
UCT: Rename uct_md_component_t to uct_component_t

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11690/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 21 workers (click for details)</summary>

**Note:** the logs will be deleted after 09-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/65565c33304d529e4bbf423e715e81b73197b044/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/b4c39755c0353421b6e69b4423a190083dfd145e/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/51e5abae090d263829ca984ad5a21fcf3addf556/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/32975eb3b6a525e184255d4f94733b5dc638371f/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/f0b1d44061e2de9b509767a814f7adfe677bb55a/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/e83a9e706d1e8ea7687aa476bacf27743f3afb07/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/a20cf706e3008b7d91a2e7dcc9322a8006f10e25/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/950fae23a1f15c1cebf64bc959db6afa15a2a424/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/3041219e5f2e6cddad2bab0561cc6a3fa694a095/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/748a30563b99f719b204c55546d7da6d13af57af/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/59e74e6008c8d368d65cb62ef66c01a8435ca505/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/f850fe26f6ae76c0e5688189226ba2c162e48580/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/c5d8d12345ba490ccc1864cd72fb4d4627ffa668/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/7c90e004cf6b6f5589d4db7eb601a31a51febfce/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/82029441defcb04dc5ad6d9b8da7e11b1ecd35eb/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/d2a0df4e3b4be0b5ba6aa0baa0eb730f6c4d9ae9/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/30cfd971154f7c1d718ebf31e41473fb8d631a60/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/418ca98574fe285506798a99613448188b84b85b/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/aa8c3d93ba377e85de11108311028e40a86eab5c/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/a5d257e82df57cb1eb528e1c673702d93ce110a7/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/36e1556040f7ad3500c638f4c9f5ffa8/raw/b05367d4c5e09bae0e7be58c764a571b3e18d87e/hpc-test-node-new_W3) | :question: UNKNOWN

File: src/uct/base/uct_component.c [hunk 1]
```diff
@@ -36,8 +36,8 @@ ucs_status_t uct_query_components(uct_component_h **components_p,
     *num_components_p = num_components;
     *components_p     = components;
 
-    ucs_list_for_each(mdc, &uct_md_components_list, list) {
-       *(components++) = mdc;
+    ucs_list_for_each(component, &uct_components_list, list) {
+       *(components++) = component;
```
Comment (evgeny-leksikov) [inline src/uct/base/uct_component.c]: need extra space to align

File: src/uct/base/uct_md.h [hunk 2]
```diff
@@ -45,14 +45,14 @@ typedef struct uct_md_registered_tl {
  * Add a transport component to a md component
  * (same transport component can be added to multiple md components).
  *
- * @param _mdc           Pointer to MD component to add the TL component to.
+ * @param _component      Pointer to MD component to add the TL component to.
```
Comment (evgeny-leksikov) [inline src/uct/base/uct_md.h]: alignment

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11692/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 09-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/580eb22e824afad65d89fee3c8f01ad47c425e81/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/6d84b32324b9785127f63b4fdd538988cbdbcedc/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/480f077d1d9737488c61f7d6f01d0509a4006aef/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/7702bb11f72588fee216107d0f3654506c01d06c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/b5f30ae45d27928504eef27e8d7156cc4c598593/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/6826974b8d544fa2f561f81872e33fcb3be6393d/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/02726492aca9c96a10da0d4e9086ec0abd74b95b/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/27eed8f96d1352c293ab4389db252919f80bdd91/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/d973573c5c2f2a3072567ebdee3f6e6f872b070c/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/a85810ea4279a1a88bc3ccf751d62d52e1e8e063/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/1a31779389877717ae8905f48d63485439af63b8/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/f84ae70bb03fac89cd660493f0d480b5950bb70b/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/24444ca27cfef4d35f9ecc4f85d3698137a85679/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/00b4f8aa957bba0b5d7604630eaa1872e6c12182/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/26ee85970be125c589ed99e3f74a88563dc78be5/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/6507badf70a4d12f02263808a26bfa379f124c79/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/85f1a63ec7038e39bc1f58c0ebf645f1dbff13c9/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/995e73c1f5a726cc768fa4384bb2c463be524ab7/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/b54a80a133368ebd8af41cd90410c510c25e1482/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/c9caf547c3209f7426f165c16da4ec343c41437c/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/b1c5a60fabe90ab77d01cf08afa325da1f2c1178/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/ee149ce7c7f8040808d65a52cf9dddc31db6558f/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/919afc4186378f63cb68ce6ed9cc8aa527601227/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/cab126bb6af55382732848c51909d8871a2886b3/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/bdc380d2cbe0311b0f11ac58571eb526a39dcce9/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/b789e072c0525a636b84e2037e4132b8f0da1791/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/76deb5f307f24ac40cfd71f4c48121ab9870e003/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/20227694769c9e25d86a8a1a6f77e2d5cd6e4f6e/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/e04da5817855d44c94328a632c0cf9ba/raw/5566705663868ee8691108ebe119fa56015e10ba/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11700/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 09-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/5d21660d7ccf4047b498f2a215844e88f5a682b3/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/08f06a294fba96dc7a13212388f5e66df7c7357a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/74f295c68a53b6bdbba6da66169395b674b16b52/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/1f6733705c59149a2c1f5366c30bf0f46aa61100/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/797a6313f73c1b3d2e1d74e6fe7ac76623a14994/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/eed6a3ec50922eac583fd55cc078bb8b86d03011/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/2e058d5a02b2c3b79a110db404d735e32dba8417/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/f378464db8a7dc111a12c5a3222aade16e36bce0/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/0dbc9089c21d04966359437f7971833770030d01/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/a8a400be0ba499677e63d27dca9897b6e02c2a9c/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/ab925d4e479cc1ea2cb1dd3d87da3f17dfbbbe4a/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/1f1ab1b1291dd7f3a2f7d9b9d8bf888eb2ec8742/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/6ff6c89e4dca16a43c4c6b8ed44f9cb1ddcbfba2/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/ba229b36da7f40bc94fc55d981abaf2bc376b4c0/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/ef2a1e0ff86577f2edea4b6d29fa339512252588/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/ed6b2f4f7aa6dbb6f2176649385c76d1144524ce/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/960fdf960f8a5f1a4d7298dfdc328827f33c1145/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/dc7c6e72b8207dbf9d2d6c3718dc5c63273c52a6/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/dbd59b5f1b819912fa1f717054019ac0e602f3c0/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/9fb2006ed25942e6e3a27cabd8334b1efecd5a61/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/14fa6c3a89a86b0a78f1dfd118d332cc5efebb8e/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/968a749595eb99070ce2ecf45eee24dc6c61199f/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/983e9b375201b04b01c074b9aea5df80973c9e41/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/cbcd9064365693a23fc222638e42f500b84d648e/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/8bea67bf3243991b851a8a16127c65a9ef0555d8/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/5b065fe27ee6494077955f007a1a536025e75a66/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/f9f87ce0a808fab7d89b746f7e90b7378ea8983d/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/cb0ca4aa5a313d10cce653de39f712790dcd567d/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/04a562f5270e9e82eae44e5a8441ec6b/raw/0798925fa0c6d8e6166d0b0090101f15cd7f4c81/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/uct/base/uct_md.h [hunk 3]
```diff
@@ -45,14 +45,14 @@ typedef struct uct_md_registered_tl {
  * Add a transport component to a md component
  * (same transport component can be added to multiple md components).
  *
- * @param _mdc           Pointer to MD component to add the TL component to.
+ * @param _component     Pointer to MD component to add the TL component to.
```
Comment (alinask) [inline src/uct/base/uct_md.h]: minor: is this still considered an MD component then? (wrt to the doc comment)

Comment (alinask) [reply src/uct/base/uct_md.h]: @yosefe

**Pull request merged**